test/memory/sungeun/refCount/domainMaps.lm-numa.good

```diff
@@ -77,4 +77,4 @@ Create domain and array:
 	0 bytes leaked
 	0 private objects leaked
 total:
-	1104 bytes leaked
+	0 bytes leaked
```

Comment:
I'd be fine doing that w.r.t. the question as to whether we can do this for all *.good files.  But it seems like we could retire this one in favor of the generic one now?  (unless I missed something in my visual diff -- it looked like both are now leak-free?)